### DISCUSSION TITLE
badgerOptions.EventLogging undefined (type badger.Options has no field or method EventLogging

### DISCUSSION THREAD

**Discussion by aliluyala** [projectdiscovery/dnsx#109] [Q&A] [answered]
**Describe the bug**
installing bug:

go get -v github.com/projectdiscovery/dnsx/cmd/dnsx
github.com/projectdiscovery/hmap/store/disk
# github.com/projectdiscovery/hmap/store/disk
/root/go/src/github.com/projectdiscovery/hmap/store/disk/badger.go:22:15: badgerOptions.EventLogging undefined (type badger.Options has no field or method EventLogging)

**Dnsx version**
Please share the version of the dnsx you are running with `dnsx -version` 

**Screenshot of the error or bug**
![image](https://user-images.githubusercontent.com/24489214/134795801-e0e10e79-1595-4a9f-8dde-e5e7d19986c2.png)

**Comment by ehsandeep** [comment, marked as answer]
@aliluyala try with `go install -v github.com/projectdiscovery/dnsx/cmd/dnsx@latest`
